plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -388,11 +370,25 @@ def __init__(self, *args, **kwargs):
         self.raw_xml = ''
 
     @property
+    def relative_path(self):  # TODO: what is this used for?
```

Comment:
I'm pretty sure this used to be the path where the file was saved in ``/var/lib/pulp/content/<type>/``, but that is no longer the case. This can probably be removed.